pylua/test/test_timer.py

```diff
@@ -70,3 +71,20 @@ def test_reset_timer(self):
         assert self._task_is_run is True, "Reset doesn't work"
         assert (timeit.default_timer() - reseted) > (
             pylua.timer.msec_to_sec(interval)), "Timer doesn't work correctly"
+
+    def _multishot_task(self):
+        self._shot_counter = self._shot_counter + 1
+
+    def test_multishot_timer(self):
+        timer = pylua.timer.PyTimer(self._multishot_task, interval=100, multishot=True)
```

Comment:
@OHerasym, please, don;t use big delays in tests. I think 30 ms interval should be enough. Timeout might be even bigger, since this timeout should never elapse.